decidim-core/app/packs/stylesheets/decidim/modules/_typography.scss

```diff
@@ -1,338 +0,0 @@
-  margin-top: -3rem;
-}
-
-.heading2 {
-  @include heading(h2);
-}
-
-.heading3 {
-  @include heading(h3);
-}
-
-.heading4 {
-  @include heading(h4);
-}
-
-.heading5 {
-  @include heading(h5);
-}
-
-.heading6 {
-  text-transform: uppercase;
-  letter-spacing: 0.03em;
-  font-weight: 600;
-
-  @include heading(h6);
-}
-
-.heading-small {
-  font-size: 1rem;
-  line-height: $header-lineheight;
-}
-
-hr {
-  width: 50%;
-
-  &.reset {
-    width: 100%;
-  }
-}
-
-/* New typographic styles */
-
-// Introduced because .section-heading has a position: relative
-// that breaks when there are two headings in the same section
-// This should be revisited in the redesign.
-
-.heading-section-heading {
-  margin-bottom: 1rem;
-  font-weight: 600;
-  font-size: 1.125em;
-  letter-spacing: 0.05em;
-  text-transform: uppercase;
-
-  &::before {
-    content: "";
-    display: inline-block;
-    width: 4px;
-    height: 1.35em;
-    background-color: var(--primary);
-    margin-right: 0.5rem;
-    margin-bottom: -0.5rem;
-    vertical-align: top;
-  }
-
-  &.collapse {
-    margin: 0;
-  }
-
-  span {
-    font-weight: normal;
-  }
-}
-
-.section-heading {
-  position: relative;
-  margin-bottom: 1rem;
-  font-weight: 600;
-  font-size: 1.125em;
-  letter-spacing: 0.05em;
-  text-transform: uppercase;
-
-  &::before {
-    content: "";
-    display: inline-block;
-    width: 4px;
-    height: 1.35em;
-    background-color: var(--primary);
-    margin-right: 0.5rem;
-    margin-bottom: -0.5rem;
-    vertical-align: top;
-  }
-
-  &.collapse {
-    margin: 0;
-  }
-
-  span {
-    font-weight: normal;
-  }
-}
-
-.mini-title {
-  color: $muted;
-  font-size: 0.9rem;
-  letter-spacing: 0.01em;
-  text-transform: uppercase;
-  font-weight: 600;
-  margin-bottom: 0;
-
-  &__strong {
-    color: $body-font-color;
-    font-size: 1.2rem;
-    font-weight: 800;
-  }
-
-  &__strong--highlight {
-    font-size: 1.4rem;
-  }
-}
-
-.data-title {
-  &__over {
-    color: $muted;
-    font-size: 0.9rem;
-    letter-spacing: 0.01em;
-    text-transform: uppercase;
-    font-weight: 800;
-    margin-bottom: 0.5rem;
-  }
-
-  &__main {
-    font-size: 2rem;
-    line-height: 1;
-    letter-spacing: 0.01em;
-    text-transform: uppercase;
-    font-weight: 800;
-    margin-bottom: 0;
-  }
-
-  &__sub {
-    color: $muted;
-    font-size: 0.9rem;
-    letter-spacing: 0.01em;
-    text-transform: uppercase;
-    line-height: 1;
-  }
-}
-
-.page-title-wrapper {
-  text-align: center;
-}
-
-.page-title {
-  margin-bottom: 3rem;
-
-  a:hover {
-    text-decoration: underline;
-  }
-}
-
-.text-highlight {
-  margin-bottom: 0;
-  color: $white;
-  text-shadow: 0 0 10px rgba(black, 0.8);
-
-  &.heading1 {
-    font-weight: 800;
-  }
-
-  > a {
-    color: inherit;
-
-    &:hover {
-      color: var(--primary);
-    }
-  }
-}
-
-// text status-color
-$map: map-merge(
-  $foundation-palette,
-  (
-    muted: $muted
-  )
-);
-
-@each $key, $value in $map {
-  .text-#{$key} {
```

Comment:
I see lots of references to these classes (text-primary, bg-success, etc), so I'd expect them to see them defined (or come from Tailwdind?) but I can't seem to find the definition nor the documentation... Can you guys point me out where's that?

Reply:
Hugo is not working this week and took me a while to find out, but Tailwind themes seem to define dynamically classes for bg and text prefixes: https://tailwindcss.com/docs/background-color#using-custom-values

I've tested it in our staging, if you apply some random classes such as `text-primary` or `bg-primary` to an element the styles are defined.

Reply:
Ah great, that's the documentation section that I was looking for. Thanks!